src/test/java/org/gridsuite/modification/server/ModificationControllerTest.java

```diff
@@ -217,8 +213,10 @@ void testEquipmentIdNonNull() {
 
     @Test
     void testNetworkNotFound() throws Exception {
-        mockMvc.perform(post(URI_NETWORK_MODIF_BAD_NETWORK)
-            .content(objectWriter.writeValueAsString(LoadCreationInfos.builder().equipmentId("id").build()))
+        ModificationApplicationContext applicationContext = new ModificationApplicationContext(NOT_FOUND_NETWORK_ID, NetworkCreation.VARIANT_ID, TEST_REPORT_ID, UUID.randomUUID());
```

Comment:
Add a method to generate the json body from a modifciation info, networkuuid, ...)